index.html

```diff
@@ -8,17 +8,25 @@
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.3.0/css/all.min.css"/>
 </head>
-<body>
-    <header class="navigation">
-        <nav class="header">
-            <a class="logo" href="#">My Logo</a>
-            <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
-        </nav>
-    </header>
- <section class="master-setion">
+<body> 
+        <header class="navigation" >
+        <div class="content">
+            <nav class="header">
+                <a class="logo" href="#">My Logo</a>
+                <a href="#" class="menu"><img src="img/menu.png" alt=""></a>
+            </nav>
+        </div>
+        </header>
+   
+    
+
 
     <section class="bio-section">
+
+
+        
         <div class="bio">
+        <div class="content">
             <h1 class="bio-header">I'm Alexandra. Glad to see you!</h1>
```

Comment:
I think it would be nice if you use your name in this part. :D
Just to start making it more personalized.